
 This patch eliminates spurious output to the console during a VerifyVolume
 job that contains encrypted data.  
 It fixes bug #1024.

 Apply this patch to Bacula version 2.2.6 and previous 2.2.x versions with:

 cd <bacula-source>
 patch -p0 <2.2.6-verify-vol.patch
 ./configure <your-options>
 make
 ...
 make install


Index: src/filed/verify_vol.c
===================================================================
--- src/filed/verify_vol.c	(revision 6010)
+++ src/filed/verify_vol.c	(working copy)
@@ -211,19 +211,6 @@
          }
          break;
 
-      /* Data streams to ignore */
-      case STREAM_ENCRYPTED_SESSION_DATA:
-      case STREAM_FILE_DATA:
-      case STREAM_SPARSE_DATA:
-      case STREAM_WIN32_DATA:
-      case STREAM_WIN32_GZIP_DATA:
-      case STREAM_GZIP_DATA:
-      case STREAM_SPARSE_GZIP_DATA:
-      case STREAM_SIGNED_DIGEST:
-
-        /* Do nothing */
-        break;
-
       case STREAM_MD5_DIGEST:
          bin_to_base64(digest, sizeof(digest), (char *)sd->msg, CRYPTO_DIGEST_MD5_SIZE, true);
          Dmsg2(400, "send inx=%d MD5=%s\n", jcr->JobFiles, digest);
@@ -256,9 +243,10 @@
          Dmsg2(20, "bfiled>bdird: SHA512 len=%d: msg=%s\n", dir->msglen, dir->msg);
          break;
 
+      /* Ignore everything else */
       default:
-         Pmsg2(0, "None of above!!! stream=%d data=%s\n", stream,sd->msg);
          break;
+
       } /* end switch */
    } /* end while bnet_get */
    set_jcr_job_status(jcr, JS_Terminated);
